### PR TITLE
Update scientific literacy chapter with revisions and extensions

### PR DESCRIPTION
**Pull request by ekormann**
Integrate major revisions and extensions into section on scientific literacy together with extended literature
#33

### CODE REVIEW

File: sections/3_societal_impact/scientific_literacy.qmd [hunk 1]
```diff
@@ -18,30 +18,37 @@ affiliations:
...
 
-First, in a view proposed by [@laugksch2000] and supported by [@roberts2007], there are at least three stakeholder groups engaged with scientific literacy: (1) sociologists, (2) public opinion researchers, (3) science educators. All three stakeholder groups approach the topic with their own goals and justifications for why scientific literacy matters. Consequently, they also employ vastly different methods to measure the concept, ranging from in-depth interviews to representative population samples, to assessments of students’ competencies.
+First, in a view proposed by @laugksch2000 and supported by @roberts2007, there are at least three stakeholder groups engaged with scientific literacy: (1) sociologists, (2) public opinion researchers, (3) science educators, including those involved in science communication. All three stakeholder groups approach the topic with their own goals and justifications for why scientific literacy matters. Consequently, they also employ different methods to measure the concept, ranging from in-depth interviews to representative population samples, to assessments of students’ competencies
```
Comment (tklebel) [inline sections/3_societal_impact/scientific_literacy.qmd:42]: You lost the trailing dot in the last sentence.

File: sections/3_societal_impact/scientific_literacy.qmd [hunk 2]
```diff
@@ -18,30 +18,37 @@ affiliations:
...
+Third, @roberts2007 provides a useful distinction between two “Visions” of scientific literacy. Here the term “Vision” is broader than a mere definition and represents an ideal type in the Weberian sense—an intentionally accentuated construct designed to highlight its core features while serving as a conceptual tool for analysis. Vision I in Roberts’ terms is concerned with literacy or knowledgeability within science, that is, it is targeted at an understanding of scientific products (publications, datasets, claims) and processes [@roberts2007, p. 730]. In contrast, Vision II is targeted at situations where scientific knowledge can aid citizens in their daily lives: “At the extreme, this vision can be called literacy (again, read thorough knowledgeability) about science-related situations in which considerations other than science have an important place at the table” [@roberts2007, p. 730]. As an example, this vision is reflected in how the OECD defines scientific literacy in PISA [@roberts2007, p. 766]:
```
Comment (tklebel) [inline sections/3_societal_impact/scientific_literacy.qmd:46]: The italics for "thoroguh knowledgability" where lost. Please check the original version.

Comment (tklebel) [reply sections/3_societal_impact/scientific_literacy.qmd:46]: This is the original: 

```
read *thorough knowledgeability*) *about science-related situations*
```

**Review by tklebel — CHANGES_REQUESTED** [review]
Thanks Eva! There are a small minor changes to finish this up.

**Comment by tklebel** [comment]
Hi @vtraag this should be good to merge now. Maybe have another look - I just looked at the text, not sure whether there is anything else you usually check.

**Review by vtraag — APPROVED** [review]
Looks good to me as well, thanks @ekormann ! I've made some minor changes (replacing quotes essentially) and cleaning up the bibtex. Assuming you have no problems with those changes, we can get this merged. I'll merge it later today, unless I hear from you.

**Pull request merged**